doc/sphinx/source/esmvaldiag/observations.inc

```diff
@@ -1,7 +1,269 @@
+
+If your run was successful, one or more NetCDF files are produced in your
+output directory.
+
+6. Naming convention of the observational data files
+====================================================
+
+For the ESMValTool to be able to read the observations from the NetCDF file,
+the file name needs a very specific structure and order of information parts
+(very similar to the naming convention for observations in ESMValTool
+v1.0). The file name will be automatically correctly created if a cmorizing
+script has been used to create the netCDF file.
+
+The correct structure of an observational data set is defined in 
+``config-developer.yml``, and looks like the following:
+
+.. code-block:: console
+ 
+  OBS_[dataset]_[type]_[version]_[mip]_[short_name]_YYYYMM-YYYYMM.nc
+
+For the example of the ``CDS-XCH4`` data set, the correct structure of the 
+file name looks then like this:
+
+.. code-block:: console
+
+  OBS_CDS-XCH4_sat_L3_Amon_xch4_200301-201612.nc
+
+The different parts of the name are explained in more detail here:
+
+- OBS: describes what kind of data can be expected in the file, in this case
+  ``observations``; 
+- CDS-XCH4: that is the name of the dataset. It has been named this way for
+  illustration purposes (so that everybody understands it is the xch4 dataset
+  downloaded from the CDS), but a better name would indeed be ``ESACCI-XCH4``
+  since it is a ESA-CCI dataset; 
+- sat: describes the source of the data, here we are looking at satellite data
+  (therefore ``sat``), could also be ``reanaly`` for reanalyses;
+- L3: describes the version of the dataset:
+- Amon: is the information in which ``mip`` the variable is to be expected, and
+  what kind of temporal resolution it has; here we expect ``xch4`` to be part
+  of the atmosphere (``A``) and we have the dataset in a monthly resolution
+  (``mon``);
+- xch4: Is the name of the variable. Each observational data file is supposed
+  to only include one variable per file; 
+- 200301-201612: Is the period the dataset spans with ``200301`` being the
+  start year and month, and ``201612`` being the end year and month;
+
+.. note::
+   There is a different naming convention for ``obs4mips`` data (see the exact
+   specifications for the obs4mips data file naming convention in the
+   ``config-developer.yml`` file).
+
+7. Test the cmorized dataset
+======================================
+
+To verify that the cmorized data file is indeed correctly formatted, you can
+run a dedicated test recipe, that does not include any diagnostic, but only
+reads in the data file and has it processed in the preprocessor. Such a recipe
+is called ``recipes/examples/recipe_check_obs.yml``. You just need to add a
+diagnostic for your dataset following the existing entries.
+
+If the recipe is adjusted as outlined above, run it with the following call:
+
+.. code-block:: console
 
+  esmvaltool -c <config-user.yml> examples/recipe_preprocessor_test.yml
+  --diagnostics <dataset>
```

Comment:
Unfortunately, this command will fail if you don't have all observational datasets listed in the recipe available, even if esmvaltool will only run the selected diagnostic. You will need to comment them out (or wait for this to be improved, see issue https://github.com/ESMValGroup/ESMValCore/issues/138).

Reply:
I would suggest removing the command and replace it by a more general sentence (something like: "Only the diagnostic of interest needs to be run, the others can be commented out for testing").

Reply:
I took the command out of the text and replaced it with the sentence: "Only the diagnostic of interest needs to be run, the others should be commented out for testing."